### PR TITLE
ZMI: fixed html

### PR DESCRIPTION
**Pull request by drfho**
Hi @jhinghaus,
excuse me there were still some html problems and the check-values of dbconn were not properly stored. I fixed that now.
Thank your for the patience
f

### CODE REVIEW

File: Products/ZPsycopgDA/dtml/add.dtml [hunk 1]
```diff
@@ -62,14 +59,14 @@
     <div class="form-group row">
       <label for="check" class="form-label col-sm-4 col-md-3">Connect immediately</label>
       <div class="col-sm-5 col-md-1">
-        <input class="btn btn-primary" type="checkbox" id="check" name="check" value="YES" checked="YES" class="form-control"/>
+        <input class="btn btn-primary" type="checkbox" id="check" name="check" value="YES" />
```
Comment (jhinghaus) [inline Products/ZPsycopgDA/dtml/add.dtml:62]: Did you remove the default checkmark on purpose?

File: Products/ZPsycopgDA/dtml/add.dtml [hunk 2]
```diff
@@ -93,21 +90,21 @@
     <div class="form-group row">
       <label for="autocommit" class="form-label col-sm-4 col-md-3">Auto-commit each query</label>
       <div class="col-sm-5 col-md-1">
-        <input class="btn btn-primary" type="checkbox" id="autocommit" name="autocommit" value="YES" checked="YES" class="form-control"/>
+        <input class="btn btn-primary" type="checkbox" id="autocommit" name="autocommit" value="YES" checked="selected" />
```
Comment (jhinghaus) [inline Products/ZPsycopgDA/dtml/add.dtml:93]: `checked="checked"` would look better on a checkbox.

File: Products/ZPsycopgDA/dtml/add.dtml [hunk 3]
```diff
@@ -93,21 +90,21 @@
...
       </div>
     </div>
 
     <div class="form-group row">
       <label for="readonlymode" class="form-label col-sm-4 col-md-3">Read-only mode</label>
       <div class="col-sm-5 col-md-1">
-        <input class="btn btn-primary" type="checkbox" id="readonlymode" name="readonlymode" value="YES" checked="YES" class="form-control"/>
+        <input class="btn btn-primary" type="checkbox" id="readonlymode" name="readonlymode" value="YES" checked="selected" />
```
Comment (jhinghaus) [inline Products/ZPsycopgDA/dtml/add.dtml:100]: See above.

**Review by jhinghaus — CHANGES_REQUESTED** [review]

**Comment by jan-jockusch** [comment]
@drfho Thanks for the PR! In the mean time, we had other problems with the UI, which prompted https://github.com/perfact/ZPsycopgDA/pull/12.

I took the liberty of incorporating your improvements there.

**Comment by drfho** [comment]
Great! 
Thanks
fh

**Comment by jan-jockusch** [comment]
This has been merged as part of https://github.com/perfact/ZPsycopgDA/pull/12, so closing.

**Pull request closed without merging**